### DISCUSSION TITLE
The newest updates (03/25/2024) are fantastic!

### DISCUSSION THREAD

**Discussion by dwsiddall** [YimMenu/HorseMenu#112] [General]
I really appreciate the hard work on these updates!

When the Time & Weather gets integrated, I will be an extra happy camper!

**Comment by Rxann** [comment]
> I really appreciate the hard work on these updates!
> 
> 
> 
> When the Time & Weather gets integrated, I will be an extra happy camper!

You are welcome. I appreciate the support. There will be more to come!
